server.py

```diff
@@ -552,13 +552,16 @@ def node_info(node_class):
 
         @routes.get("/object_info")
         async def get_object_info(request):
+            folder_paths.cache_helper.clear()
+            folder_paths.cache_helper.active = True
```

Comment:
This would look cleaner if it was using a context.

Reply:
good point - applied